package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "devDependencies": {
```

Comment:
Where is the actual prettier dependency? What about scripts to actually format and check the state of the files.